addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -312,7 +312,7 @@ def _post_invoice_edi(self, invoices, test_mode=False):
     # -------------------------------------------------------------------------
 
     def _check_filename_is_fattura_pa(self, filename):
-        return re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(xml.p7m|xml))", filename)
+        return re.search("([A-Z]{2}[A-Z0-9]{2,28}_[A-Z0-9]{0,5}.(XML.P7M|XML))", filename.upper())
```

Comment:
you could do this:
```python3
return re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(xml.p7m|xml))", filename, flags=re.IGNORGECASE)
```

but I prefer this:
```python3
return re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(xml.p7m|xml|XML.P7M|XML))", filename)
```

Reply:
actually, I think
```python 3
 re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(?i:p7m\.){0,1}(?i:xml))"
``` 
is best
`?i:` tells regex that it is case insensitive

Reply:
@dbkosky It is xml.p7m:  why not (?i:xml.p7m|?i:xml) ?

Reply:
oops yes, it's actually xml.p7m then
```
 re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(?i:xml)(?i:\.p7m){0,1})"
```
.. you could do as you say as well, in that case it would only need to be `(?i:xml|xml.p7m)`